### ISSUE TITLE
Display information about running service

### ISSUE THREAD

**Issue by smaaland** [tilt-dev/tilt#6215]
## Describe the Feature You Want
It would be nice to be able to see which version (e.g. docker image tag) of the service that you're running in the sidebar. Or at least a possibility to add some kind of custom text there besides the resource name. Changing the resource name with `new_name` is not an alternative since it creates a lot more problems with helm charts and dependency chains.

We tend to work on only a few services at the time in "developer mode", and for the rest of the stack we run pre-built images with helm. But it's very difficult to see at a glance which version of a service/resource that's running.

**Comment by nicks** [comment]
we often see people do this kind of thing with UI buttons that display whatever custom data they want, here's a guide: https://docs.tilt.dev/buttons

**Comment by smaaland** [comment]
Yes, I know. The thing I'm missing is to have it (or some custom text that I can generate myself) in the listing view in the sidebar. It's kind of tedious to click through 50+ services and look through them one by one to find out which one is out of date.

**Comment by nicks** [comment]
OK. We can leave this open for suggestions of possible ways to solve this problem. i think it's unlikely we would add a feature to customize the sidebar.
